pkg/controller/pravega/bookie.go

```diff
@@ -167,14 +172,20 @@ func makeBookieVolumeClaimTemplates(spec *v1alpha1.BookkeeperSpec) []corev1.Pers
 			},
 			Spec: *spec.Storage.LedgerVolumeClaimTemplate,
 		},
+		{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: IndexDiskName,
+			},
+			Spec: *spec.Storage.IndexVolumeClaimTemplate,
+		},
 	}
 }
 
 func MakeBookieConfigMap(pravegaCluster *v1alpha1.PravegaCluster) *corev1.ConfigMap {
 	configData := map[string]string{
 		"BK_BOOKIE_EXTRA_OPTS":     "\"-Xms1g -Xmx1g -XX:MaxDirectMemorySize=1g -XX:+UseG1GC  -XX:MaxGCPauseMillis=10 -XX:+ParallelRefProcEnabled -XX:+UnlockExperimentalVMOptions -XX:+AggressiveOpts -XX:+DoEscapeAnalysis -XX:ParallelGCThreads=32 -XX:ConcGCThreads=32 -XX:G1NewSizePercent=50 -XX:+DisableExplicitGC -XX:-ResizePLAB\"",
 		"ZK_URL":                   pravegaCluster.Spec.ZookeeperUri,
-		"BK_useHostNameAsBookieID": "true",
+		"BK_useHostNameAsBookieID": "false",
```

Comment:
Can I ask why are we changing this back to `false`?

Reply:
We changed this to `true` when https://github.com/pravega/pravega/pull/3357 was merged, however, `pravega/pravega:latest` is still based on BK 4.5, so we will have a regression (https://github.com/pravega/pravega-operator/issues/116) if we keep it `true`. I think we should set it to `false` until Pravega 0.5 is released. 

@Tristan1900 any thoughts?

Reply:
You are right. Thanks for the explanation.